src/UI Component/User/loanapplication/employerinfo.js

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+
+const LoanInformation = ({ formData, handleChange }) => {
+  return (
+    <form className='flex text-sm flex-col'>
+        <label className="text-sm text-black font-bold mb-2">Company Name</label>
+            <input
+                type="text"
+                name="companyName"
+                value={formData.companyName}
+                onChange={handleChange}
+                className="border  text-blue-900 rounded p-2 mb-4"
+                placeholder="ABC Coporation"
+                
+                
+            />
+ 
+
+ <label className="text-sm text-black font-bold mb-2">Job Title</label>
+            <input
+                type="text"
+                name="jobTitle"
+                value={formData.jobTitle}
+                onChange={handleChange}
+                className="border text-black rounded p-2 mb-4"
+                placeholder="Banker"
+               
+            />
+            <label className="text-sm font-bold text-black mb-2">Company's Address</label>
+            <input
+                type="text"
+                name="companyAddress"
+                value={formData.companyAddress}
+                onChange={handleChange}
+                className="border rounded  text-blue-900 p-2 mb-4"
+                placeholder="123 Main St, Ciyt,Country"
+                
+            />
+  <label className="text-sm font-bold  text-black mb-2">
+      Duration of work
+    </label>
+    <select
```

Comment:
You are using html select twice in this component. 

Create a reusable component that dynamically shows the options, handles the change and return the value as it's selected. 

Please avoid code repetition in any where possible